linux/CMakeLists.txt

```diff
@@ -1,26 +1,30 @@
 #Copyright (C) Microsoft Corporation. All rights reserved.
 
 set(pal_linux_h_files
+    ${pal_common_h_files}
     inc/refcount_os.h
 )
 
 set(pal_linux_c_files
+    ${pal_common_c_files}
     src/interlocked_linux.c
     src/lock_pthreads.c
     src/platform_linux.c
     src/threadapi_pthreads.c
     src/uniqueid_linux.c
     src/sync_linux.c
+    src/file_linux.c
 )
 
 FILE(GLOB pal_linux_md_files "devdoc/*.md")
 SOURCE_GROUP(devdoc FILES ${pal_linux_md_files})
 
 include_directories(${CMAKE_CURRENT_LIST_DIR}/inc)
+include_directories(../common/inc)
 
 add_library(pal_linux ${pal_linux_h_files} ${pal_linux_c_files} ${pal_linux_md_files})

```

Comment:
>pal_linux_md_files [](start = 66, length = 18)

Might be nice to also add pal_common_md_files

Reply:
coming up in next iteration

---
In reply to: [451914064](https://github.com/Azure/azure-c-pal/pull/7#discussion_r451914064) [](ancestors = 451914064)